src/core/Link/Link.baseStyles.ts

```diff
@@ -31,3 +31,24 @@ export const externalStyles = withSuomifiTheme(
     }
   `,
 );
+
+export const skipStyles = withSuomifiTheme(
```

Comment:
I'd rename this to skipLinkStyles. I first thought this means disabling some Link styles when using the skip variant.

Reply:
This is now changed.